gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -88,6 +88,10 @@
   /** Default setting for GCS HTTP request headers. */
   public static final ImmutableMap<String, String> HTTP_REQUEST_HEADERS_DEFAULT = ImmutableMap.of();
 
+  /** Default setting for impersonation service accounts. */
+  public static final ImmutableMap<String, String> IMPERSONATION_SERVICE_ACCOUNTS =
```

Comment:
Please add a "DEFAULT" postfix to the variable name.

Reply:
Default will be an empty mapping and will be ignored at credential fetching time. If we actually put some default "string" then we need to actually differentiate the default string and actual user/group name which would rather be an empty check.